tests/post/features/service_configuration.feature

```diff
@@ -7,3 +7,14 @@ Feature: Cluster and Services Configurations
         And we have a 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth'
         When we update the CSC 'spec.deployment.replicas' to '3'
         Then we have '3' at 'status.available_replicas' for 'dex' Deployment in namespace 'metalk8s-auth'
+
+  Scenario: Update Admin static user password
+        Given the Kubernetes API is available
+        And we have a 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth'
+        # NOTE: Consider that admin user is the first of the userlist
+        # Update password to "new-password"
+        When we update the CSC 'spec.localuserstore.userlist.0.hash' to '$2y$14$pDe0vj917rR3XJQ5iEZvhuSGoWkZg2/qBN/mMLqwFSz9S7EYcbIpO'
+        Then the control-plane Ingress path '/oidc' is available
+        And pods with label 'app.kubernetes.io/name=dex' are 'Ready'
+        And we are not able to login to Dex as 'admin@metalk8s.invalid' using password 'password'
+        And we are able to login to Dex as 'admin@metalk8s.invalid' using password 'new-password'
```

Comment:
The corresponding methods only exist in `tests/post/steps/test_authentication.py`, I don't think it'll work.
You should either put them in `tests/conftest.py` to make these `Then` available to all your scenario, or declare them in `tests/post/steps/test_service_configuration.py`

Reply:
Right, cool pytest-bdd feature (typo = success) .... 
Anyway ok should be fixed but I don't know if my test work as he actually never run for the moment